pages/_document.js

```diff
@@ -67,8 +68,12 @@ ga('create', '${config.google.id}', 'material-ui.com');
               `,
             }}
           />
-          <NextScript />
-          <script async src="https://cdn.jsdelivr.net/docsearch.js/2/docsearch.min.js" />
+          <script defer src="https://cdn.jsdelivr.net/docsearch.js/2/docsearch.min.js" />
+          {/* Dynamic polyfill for IE 11 */}
+          <script
+            defer
+            src="https://cdn.polyfill.io/v2/polyfill.min.js?features=Array.prototype.findIndex"
```

Comment:
🙊

Reply:
Wouldn't it be better to import the polyfill in the actual file so that next.js will only use it in that chunk? RIght now this is still somewhat dangerous when people land on pages that require that polyfill.